app.js

```diff
@@ -1,10 +1,16 @@
 const express = require("express");
 const { getAllTopics } = require("./controllers/topicsController");
+const { getArticleById } = require("./controllers/articlesController");
+const { sqlErrors, customErrors } = require("./middleware/errorHandlers");
 const app = express();
 
 app.use(express.json());

```

Comment:
Please remove this until it is needed we're not dealing with any incoming data yet.